README.md

```diff
@@ -133,69 +186,150 @@ To test quipucords against virtual machines running on a cloud provider, view th
 
 # <a name="advanced"></a> Advanced Topics
 
-## Container Image
+##  Installing and running the server and database containers 
 The quipucords container image can be created from source. This quipucords repository includes a Dockerfile that contains instructions for the image creation of the server.
-You must have [Docker installed](https://docs.docker.com/engine/installation/) to create the image and run the container.  The following examples all use version `0.0.46` but any version could be used.
-
+You must have [Docker installed](https://docs.docker.com/engine/installation/) or [Podman installed](https://podman.io/getting-started/installation).
```

Comment:
It may be helpful to add a comment here stating that the examples that follow use `podman` but can be replaced with `docker`